### ISSUE TITLE
Request new pattern for Incomplete structure

### ISSUE THREAD

**Issue by LCCarmody** [obophenotype/upheno#931]
ZP, XPO use a 'incomplete structure' and now MP and HP will start using this. Need to add a pattern for this.

'has part' some
('Incomplete structure' and ('characteristic of' some Anatomical entity)
and ('has modifier' some abnormal))

Refers to https://github.com/obophenotype/human-phenotype-ontology/issues/7389
@anna-anagnostop

**Comment by LCCarmody** [comment]
Remind me how far that goes up? If it is an incomplete cochlea, what would 'characteristic of part of' mean? An incomplete 'inner ear', yes. An 'incomplete ear', I don't know.

**Comment by sbello** [comment]
Yeah, I think it would go all the way up the part tree. We had a request to see what exactly happens (see https://github.com/obophenotype/upheno/issues/925)
The spreadsheets are ready to go for this we just need to push the next steps through.
From my perspective if part of the ear is incomplete the ear is incomplete but I don't know that a clinician would agree.

**Comment by pnrobinson** [comment]
An incomplete structure of the inner ear would be regarded as an incomplete structure of the ear by the clinician, but probably no clinician would say incomplete ear for this.
